.gitlab/ci/bucket-upload.yml

```diff
@@ -462,13 +462,46 @@ upload-packs-to-xpanse-marketplace:
     - section_end "Zip Content Packs From GCS"
     - job-done
 
-force-pack-upload:
+
+force-pack-upload-xsoar:
```

Comment:
Let's call the steps `<MP>-force-pack-upload` as in the prepare steps, because in the pipeline the step names might be broken and can't be identified by the MP name.